### PR TITLE
PYTHON-3119 getMore helper should explicitly send inherited comment

### PR DESCRIPTION
**Pull request by juliusgeo**

### CODE REVIEW

**Review by blink1073 — APPROVED** [review]
LGTM!

File: pymongo/command_cursor.py [hunk 1]
```diff
@@ -207,7 +209,6 @@ def _refresh(self):
         """
         if len(self.__data) or self.__killed:
             return len(self.__data)
-
```
Comment (ShaneHarvey) [inline pymongo/command_cursor.py:210]: Uneeded?

File: test/test_examples.py [hunk 2]
```diff
@@ -1201,6 +1201,7 @@ def test_versioned_api(self):
         client = MongoClient(uri, server_api=ServerApi("1", deprecation_errors=True))
         # End Versioned API Example 4
 
+    @unittest.skip("PYTHON-3167 count has been added to API version 1")
```
Comment (ShaneHarvey) [inline test/test_examples.py:1204]: How come this is showing up in the diff?

File: test/test_monitoring.py [hunk 3]
```diff
@@ -239,6 +239,7 @@ def _test_find_options(self, query, expected_cmd):
             # Exhaust the cursor to avoid kill cursors.
             tuple(cursor)
 
+    @client_context.require_version_min(4, 4, 4)
```
Comment (ShaneHarvey) [inline test/test_monitoring.py:242]: Why is this needed?

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]

File: test/test_monitoring.py (hunk 3 above)
Comment (juliusgeo) [reply test/test_monitoring.py:242]: "Comment" is not supported on "getMore" commands below 4.4.4 IIUC

File: pymongo/command_cursor.py (hunk 1 above)
Comment (juliusgeo) [reply pymongo/command_cursor.py:210]: Yup, sorry about that. Restored.

File: pymongo/message.py [hunk 4]
```diff
@@ -467,13 +472,13 @@ def use_command(self, sock_info):
     def as_command(self, sock_info):
         """Return a getMore command document for this query."""
         # See _Query.as_command for an explanation of this caching.
+
```
[the remark this answers is not in the document]
Comment (juliusgeo) [reply pymongo/message.py:475]: Fixed.

File: .gitignore [hunk 5]
```diff
@@ -15,3 +15,4 @@ pymongo.egg-info/
 .tox
 mongocryptd.pid
 .idea/
+.nova/
```
[the remark this answers is not in the document]
Comment (ShaneHarvey) [reply .gitignore:18]: It's fine here, I'm curious what IDE?

File: test/test_monitoring.py (hunk 3 above)
Comment (ShaneHarvey) [reply test/test_monitoring.py:242]: Doesn't that make this PR a major breaking change?

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]

File: test/test_monitoring.py (hunk 3 above)
Comment (juliusgeo) [reply test/test_monitoring.py:242]: I checked the [getMore docs](https://docs.mongodb.com/manual/reference/command/getMore/) and it appears that comment is not supported below 4.4.4. So yes, this would be a major breaking change.

**Comment by ShaneHarvey** [comment]
Changes LGTM so far. Can you implement the tests in https://github.com/mongodb/specifications/pull/1161?

**Review by ShaneHarvey — APPROVED** [review]
